frontend/src/pages/ResourcesDashboard/__tests__/index.js

```diff
@@ -0,0 +1,240 @@
+  {
+    name: 'Resource URL 4',
+    url: '',
+    reportCount: 12,
+    participantCount: 11,
+    recipientCount: 10,
+  },
+];
+
+const lastThirtyDays = formatDateRange({
+  lastThirtyDays: true,
+  forDateTime: true,
+});
+const allRegions = 'region.in[]=1&region.in[]=2';
+const lastThirtyDaysParams = `startDate.win=${encodeURIComponent(lastThirtyDays)}`;
+
+const regionInParams = 'region.in[]=1';
+
+describe('Resources Dashboard page', () => {
+  afterEach(() => fetchMock.restore());
+  const renderResourcesDashboard = (user) => {
+    render(
+      <UserContext.Provider value={{ user }}>
+        <Router history={history}>
+          <ResourcesDashboard user={user} />
+        </Router>
+      </UserContext.Provider>,
+    );
+  };
+
+  it('renders correctly', async () => {
+    // Page Load.
+    fetchMock.get(`${resourceListUrl}?${allRegions}&${lastThirtyDaysParams}`, resourceListResponse);
+    fetchMock.get(`${resourceOverviewUrl}?${allRegions}&${lastThirtyDaysParams}`, resourcesOverview);
+
+    // Region 1.
+    fetchMock.get(`${resourceListUrl}?${regionInParams}`, resourceListResponseRegionOne);
+    fetchMock.get(`${resourceOverviewUrl}?${regionInParams}`, resourcesOverviewRegionOne);
+
+    const user = {
+      homeRegionId: 14,
+      permissions: [{
+        regionId: 1,
+        scopeId: SCOPE_IDS.READ_ACTIVITY_REPORTS,
+      }, {
+        regionId: 2,
+        scopeId: SCOPE_IDS.READ_ACTIVITY_REPORTS,
+      }],
+    };
+
+    renderResourcesDashboard(user);
+    expect(await screen.findByText(/resources dashboard/i)).toBeVisible();
+
+    // Resource List (initial).
+    expect(screen.getByText(/resources in activity reports/i)).toBeInTheDocument();
+    expect(screen.getByRole('columnheader', { name: /resource/i })).toBeInTheDocument(3);
+    expect(screen.getByRole('columnheader', { name: /number of activities/i })).toBeInTheDocument();
+    expect(screen.getByRole('columnheader', { name: /number of participants/i })).toBeInTheDocument();
+    expect(screen.getByRole('columnheader', { name: /number of recipients/i })).toBeInTheDocument();
+    expect(screen.getByRole('cell', { name: /resource one/i })).toBeInTheDocument();
+    expect(screen.getByRole('cell', { name: /4/i })).toBeInTheDocument();
+    expect(screen.getByRole('cell', { name: /6/i })).toBeInTheDocument();
+    expect(screen.getByRole('cell', { name: /2/i })).toBeInTheDocument();
+    expect(screen.getByRole('cell', { name: /resource two/i })).toBeInTheDocument();
+    expect(screen.getByRole('cell', { name: /2/i })).toBeInTheDocument();
+    expect(screen.getByRole('cell', { name: /8/i })).toBeInTheDocument();
+    expect(screen.getByRole('cell', { name: /2/i })).toBeInTheDocument();
+
+    // Overview (initial).
+    expect(screen.getByText(/95.24%/i)).toBeInTheDocument();
+    expect(screen.getByText(/^[ \t]*Recipients rec'd resources\r?\n?[ \t]*220 of 231/i)).toBeInTheDocument();
+    expect(screen.getByText(/94.81%/i)).toBeInTheDocument();
```

Comment:
Because they are broken up weirdly by the tooltips, these tests need to be a little trickier (and broken into 2)

Like so:
```js
expect(screen.getByText(/95.24%/i)).toBeInTheDocument();
expect(screen.getByText(/recipients rec'd resources/i, { selector: 'span:not(.sr-only)' })).toBeVisible();
```